pyiron_contrib/__init__.py

```diff
@@ -58,6 +58,7 @@
         "PigletMD": "pyiron_contrib.atomistics.ipi.ipi_jobs",
         "FitsnapJob": "pyiron_contrib.atomistics.fitsnap.job",
         "QuasiHarmonicApproximation": "pyiron_contrib.atomistics.atomistics.master.qha",
+        "MDAnalysis": "pyiron_electrochemistry.md_analysis"
```

Comment:
You can import `from pyiron_base import JOB_CLASS_DICT` in `pyiron_electrochemistry` and then just load the module there: 
```
JOB_CLASS_DICT.update({"MDAnalysis": "pyiron_electrochemistry.md_analysis"})
```

Reply:
Okay that worked. It was added in the __init_ file and not in the md_analysis class.